docker-cronjob/README.md

```diff
@@ -0,0 +1,89 @@
+# Docker Image Pruning Cron Job for Dev server
+
+This README provides instructions for setting up a cron job to regularly prune old Docker images on a Dev server. Pruning Docker images helps to manage disk space effectively by removing unused images.
+
+## Prerequisites
+
+- SSH access to the Dev server.
+
+## Steps to Set Up the Cron Job
+
+### 1. Access the Dev Server
+
+SSH into your Dev server:
+
+```sh
+ssh azureuser@ui-1.dev.codeforafrica.org
+```
+
+### 2. Create the Pruning Script
+
+Create a script to prune Docker images and make it executable.
+
+Create the Script
+Create a new file for the pruning script:
+
+```sh
+sudo nano prune_docker_images.sh
+```
+
+Add the following content to the script:
+
+```sh
+#!/bin/bash
+sudo docker image prune -f
+```
```

Comment:
Do we need this section? This script already exists, no?

Reply:
Well, that was for reusing the script somewhere else. However, yes I agree with you @kilemensi